console/src/main/java/com/alibaba/nacos/console/controller/UserController.java

```diff
@@ -146,6 +155,23 @@ public Object updateUser(@RequestParam String username, @RequestParam String new
         
         return new RestResult<>(200, "update user ok!");
     }
+
+    private boolean hasPermission(String username, HttpServletRequest request) {
+        if (!authConfigs.isAuthEnabled()) {
+            return true;
+        }
+        if (request.getAttribute(RequestUtil.NACOS_USER_KEY) == null) {
```

Comment:
use `com.alibaba.nacos.common.utils.StringUtil#isBlank ` will be better, I think

Reply:
use util is better , but it is an object.  I will use `com.alibaba.nacos.common.utils.Objects#isNull`

Reply:
If nacos_user_key is "", whether it return false directly?